improver/ensemble_copula_coupling/ensemble_copula_coupling.py

```diff
@@ -633,7 +642,7 @@ def __init__(self, distribution="norm", shape_parameters=None):
                 require the specification of shape parameters to be able to
                 define the shape of the distribution. For the truncated normal
                 distribution, the shape parameters should be appropriate for
-                distribution constructed from the mean and standard deviation
+                distribution constructed from the location and shape parameters
```

Comment:
Maybe this sentence just needs another "a" or something? Doesn't read quite right at the moment.

Reply:
I've made a minor edit to this sentence.